### ISSUE TITLE
Accessibility: Readable elements and navigation through screen reader

### ISSUE THREAD

**Issue by jansche** [bpetetot/conference-hall-beta#853]
Working with and on behalf of a friend with no vision, we identified some issues during a walk through of the talk submission process. Splitting these into single issues for workflow purposes.

Browser: Edge (87.0.664.41)
Screen Reader: JAWS

Problem: When navigating the site with tabulators everything seems ok, but as soon as the screen reader navigates readable elements, nearly all links on the site are split into two elements that get focus (icon and text). There should be only one item.
This issue becomes more prominent when navigating [this page](https://conference-hall.io/speaker/event/FdP7ePMBrbLq9kZd4Hye/submission) with a screen reader (arrow keys): Ordr of focus is Talk selection, Talk details, Event Submission, Done ! and then mic icon, calendar icon, paper plane icon.
The first link where this behaviour gets acknowledgable is the report a bug link, that has the GitHub icon and text, both of these get the focus og the screen reader, but only the text can be identified. The first one (icon) is empty and cannot be read out. Grouping them together makes the page easier to navigate.

**Comment by bpetetot** [comment]
Thanks for the feedback. It's very useful 🙏
I don't have Jaws or Windows to reproduce but I followed some instructions to manage correctly navigation with a screen reader. I put `role=presentation` attributes to some icons, so the screen reader will no read them anymore.
Fix: a6ff40844002947834675630f26b545dd888fcae
